R/trans.r

```diff
@@ -79,13 +82,19 @@ lines.trans <- function(x, ..., xlim) {
 
 #' @rdname trans_new
 #' @export
-as.trans <- function(x) {
+as.trans <- function(x, arg = deparse(substitute(x))) {
   if (is.trans(x)) {
-    return(x)
+    x
+  } else if (is.character(x) && length(x) >= 1) {
+    if (length(x) == 1) {
+      f <- paste0(x, "_trans")
+      match.fun(f)()
+    } else {
+      compose_trans(!!!lapply(x, as.trans))
```

Comment:
The `as.trans` step seems to be repeated here as well as at the top of `compose_trans()`. We could probably remove this one and use the one in `compose_trans()`, unless I might be missing something.